minio/helpers.py

```diff
@@ -325,8 +325,9 @@ def is_virtual_host(endpoint_url, bucket_name):
     # such buckets.
     if 'https' in parsed_url.scheme and '.' in bucket_name:
         return False
-    if 's3.amazonaws.com' in parsed_url.netloc:
-        return True
+    for host in ['s3.amazonaws.com', 'aliyuncs.com']:
```

Comment:
@laixintao if things are working, I have no issues about this change.  Can you confirm?

Reply:
I confirmed.